src/app/containers/App/App.jsx

```diff
@@ -32,6 +32,16 @@ export const App = ({ routes, location, initialData, bbcOrigin, history }) => {
 
   const isInitialMount = useRef(true);
 
+  useEffect(() => {
+    if (!isInitialMount.current && state.loading === false) {
+      const contentElem = document.querySelector('h1#content');
```

Comment:
I wouldn't scope this to h1 elements in case the ID is added to media or some other non h1 element

Reply:
I think there's potential that this code could run even when there is no route transition

Reply:
To solve this, you might to introduce specific state that won't cause a rerender when you set it back to false e.g.

```js
const shouldSetFocus = useRef(false);
const focusContentEl = () => {
  const contentEl = document.querySelector('#content');
  if (contentEl) contentEl.focus();
}

useEffect(() => {
  if (shouldSetFocus.current) {
    shouldSetFocus.current = false;
    focusContentEl();
  }
}, [state.loading]);






route.getInitialData(location.pathname)
  .then(data =>
    setState(prevState => ({
      ...prevState,
      loading: false,
      pageData: path(['pageData'], data),
      status: path(['status'], data),
      error: path(['error'], data),
    })),
  )
  .then(() => {
    shouldSetFocus.current = true;
  })
```

I'm not sure if there's a better way to do this.

Reply:
Thanks Jonathan. After running the initial approach by @greenc05 . We agreed to target only the h1 for now. h1 with id="content" to be doubly safe. These issues cater for the case you mentioned and other other edge cases. https://github.com/bbc/simorgh/issues/1524 https://github.com/bbc/simorgh/issues/1525

Reply:
OK fair enough. I'd be interested to know reasoning for that

Reply:
I believe the reasoning is so that focus is not placed on, for example, an image for cases where the first content on a page is an image and not the Headline. So as not to confuse screen reader users